doc/compiling/linux.md

```diff
@@ -71,7 +71,7 @@ Download source (this is the URL to the latest of source repository, which might
 
 Download IrrlichtMt to `lib/irrlichtmt`, it will be used to satisfy the IrrlichtMt dependency that way:
 
-    git clone --depth 1 https://github.com/minetest/irrlicht.git lib/irrlichtmt
+    git clone --depth 1 --branch $(cat misc/irrlichtmt_tag.txt) https://github.com/minetest/irrlicht.git lib/irrlichtmt
```

Comment:
```suggestion
    git clone --depth 1 --branch "$(cat misc/irrlichtmt_tag.txt)" https://github.com/minetest/irrlicht.git lib/irrlichtmt
```
Same for macos.md. This ensures that the file contents are never treated as command line arguments. It may never happen here but it's better to do it properly.

Reply:
Done.